docs/configuration.md

```diff
@@ -17,11 +19,36 @@ The configuration uses the following structure:
 
 ```js
 {
+  threshold: number;
```

Comment:
I think the description for attribute in config is needed.
Like here [IMA Docs](https://github.com/kaltura/playkit-js-ima/blob/master/docs/api.md)